swig/swig_interfaces/oc_session_events.i

```diff
@@ -25,32 +25,6 @@
 %ignore oc_session_event_cb;
 %ignore oc_session_events;
 
-%ignore oc_session_start_event;
-%rename (startEvent) jni_session_start_event;
-%inline %{
-void jni_session_start_event(oc_endpoint_t *endpoint)
-{
-#ifdef OC_TCP
-  oc_session_start_event(endpoint);
-#else /* OC_TCP */
-  OC_DBG("JNI: %s - Must build with OC_TCP defined to use this function.\n", __func__);
-#endif /* !OC_TCP */
-}
-%}
-
-%ignore oc_session_end_event;
-%rename (endEvent) jni_session_end_event;
-%inline %{
-void jni_session_end_event(oc_endpoint_t *endpoint)
-{
-#ifdef OC_TCP
-  oc_session_end_event(endpoint);
-#else /* OC_TCP */
-  OC_DBG("JNI: %s - Must build with OC_TCP defined to use this function.\n", __func__);
-#endif /* !OC_TCP */
-}
-%}
-
```

Comment:
Why these is removed ?

Reply:
only public functions should be in the JAVA wrapper, these are internal